helix-term/src/ui/editor.rs

```diff
@@ -1126,6 +1138,13 @@ impl Component for EditorView {
         };
 
         match event {
+            Event::Paste(contents) => {
+                cx.count = cx.editor.count;
```

Comment:
I felt like it made more sense to use a pending count here, though I could also see leaving it for an upcoming command.